app/Http/Controllers/StatuslabelsController.php

```diff
@@ -281,7 +281,7 @@ public function getDatatable()
             $actions .= Helper::generateDatatableButton('edit', route('statuslabels.edit', $statuslabel->id));
             $actions .= Helper::generateDatatableButton(
                 'delete',
-                route('statuslabels.destroy', ['statuslabel' => $statuslabel->id]),
```

Comment:
I think we needed this? Was getting an error on status labels page before I changed it